features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/index.jsx

```diff
@@ -840,7 +844,7 @@ class Details extends Component {
                                                 Icon={<EndpointIcon />}
                                             />
                                         )}
-                                        {!isAPIProduct && (
+                                        {!api.advertiseInfo.advertised && !isAPIProduct && (
```

Comment:
Same as above, This should be OR operator, API scope menu should be hidden when API is advertised or if it's an API product.

&& would only hide it when it's an API product and also it's advertised

Reply:
@tmkasun we decided it should be a AND condition right ?